src/train.py

```diff
@@ -310,12 +339,21 @@ def plot_metrics(metrics, class_labels, train_counts, test_counts):
         class_labels, precision, recall, f1, train_counts, test_counts, strict=False
     )
 
-    table = wandb.Table(
-        data=[list(values) for values in data],
-        columns=["label", "precision", "recall", "f1", "train_count", "test_count"],
-    )
+    # Only proceed if wandb reporting is enabled
+    if config["training_settings"]["report_to"] != "wandb":
+        # print logging metrics table to terminal
+        print("Metrics and value counts:")
+        for values in data:
+            print(values)
+        return
```

Comment:
It might be nice to do both, save having to leave the terminal even when using wandb?